.github/cilium-actions.yml

```diff
@@ -33,7 +33,7 @@ move-to-projects-for-labels-xored:
       column: "Backport done to v1.5"
 require-msgs-in-commit:
   - msg: "Signed-off-by"
-    helper: "https://docs.cilium.io/en/stable/contributing/contributing/#developer-s-certificate-of-origin"
+    helper: "http://docs.cilium.io/en/stable/contributing/development/contributing_guide/#developer-s-certificate-of-origin"
```

Comment:
Any reason you switched to `http`?

Reply:
hadn't realized I did, I followed a link from the readme which uses http, copied the whole url and pasted it....

Reply:
Could you please reverse this change and sign off your commit?